pom.xml

```diff
@@ -3,20 +3,20 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.26</version>
+    <version>2.33</version>
   </parent>
 
   <groupId>org.jenkins-ci.modules</groupId>
   <artifactId>windows-slave-installer</artifactId>
-  <version>1.10-SNAPSHOT</version>
+  <version>1.9.1-SNAPSHOT</version>
```

Comment:
@oleg-nenashev Could I ask why it goes from 1.10 to 1.9.1? Should it not be 1.10.1?